app/src/pages/inside/launchesPage/launchSuiteGrid/hamburger/hamburger.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+import classNames from 'classnames/bind';
+import { GhostButton } from 'components/buttons/ghostButton';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+import styles from './hamburger.scss';
+
+const cx = classNames.bind(styles);
+
+export class Hamburger extends Component {
+  static propTypes = {
+    mode: PropTypes.string.isRequired,
+    onAction: PropTypes.func,
+  };
+  static defaultProps = {
+    onAction: () => {},
```

Comment:
I think it's better to have different handlers for all actions instead of the universal one